module/lua/setjmp/setjmp_x86_64.S

```diff
@@ -23,18 +23,27 @@
  * Copyright (c) 1992, 2010, Oracle and/or its affiliates. All rights reserved.
  */
 
- #ifdef _WIN32
+#ifdef _WIN32
```

Comment:
We can add a win/asm_linkage.h to assist us. We have a windows asm_linkage, but they do need to be separated by platform.